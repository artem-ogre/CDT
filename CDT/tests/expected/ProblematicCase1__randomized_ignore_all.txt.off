OFF
11 17 0
-7.711142542794271 -9.070832376358855 0
23.71114254279427 -9.070832376358855 0
8 18.14166475271771 0
0 0 0
6 2 0
8 1 0
10 2 0
16 0 0
10 -2 0
8 -1 0
6 -2 0
3 3 0 10
3 3 5 4
3 2 0 3
3 7 3 9
3 7 9 8
3 2 3 4
3 8 9 10
3 3 7 5
3 9 3 10
3 5 7 6
3 1 2 7
3 1 10 0
3 1 8 10
3 2 6 7
3 1 7 8
3 6 4 5
3 6 2 4
