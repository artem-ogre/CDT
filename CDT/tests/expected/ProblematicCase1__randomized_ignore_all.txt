17
0 1 10   4294967295 5 2
0 3 2   2 6 4294967295
0 10 3   0 12 1
1 2 7   4294967295 8 4
1 7 8   3 15 5
1 8 10   4 16 0
2 3 4   1 9 7
2 4 6   6 13 8
2 6 7   7 14 3
3 5 4   10 13 6
3 7 5   11 14 9
3 9 7   12 15 10
3 10 9   2 16 11
4 5 6   9 14 7
5 7 6   10 8 13
7 9 8   11 16 4
8 9 10   15 12 5

1
3 7

0

0
